std_msgs/Header header
geometry_msgs/Pose pose
std_msgs/Bool near
std_msgs/Bool see_anode
std_msgs/Bool see_cathode